db/development_recreate.sh

```diff
@@ -0,0 +1,5 @@
+RAILS_ENV=development bundle exec rake db:drop
+RAILS_ENV=development bundle exec rake db:create
+RAILS_ENV=development bundle exec rake db:schema:load
+RAILS_ENV=development bundle exec rake db:migrate
+RAILS_ENV=development bundle exec rake db:seed
```

Comment:
This is a good shortcut to make for yourself, but it shouldn't be added to the repo.  Particularly not as a shell script.